ro-crate-metadata.json

```diff
@@ -257,7 +254,8 @@
             "@id": "https://nf-co.re/",
```

Comment:
I guess also this section should be genomic-medicine-sweden, as it is `"@type": "Organization"`, which I assume is the hosting organization?